website/pages/blog/announcing-graphql-network-inspector.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Announcing GraphQL Network Inspector
+tags: [graphql, graphql-network-inspector]
+authors: [warrenjday]
+date: 2023-02-10
+description: A better way to debug network traffic with GraphQL
+image: /blog-assets/announcing-graphql-network-inspector/cover.png
+thumbnail: /blog-assets/announcing-graphql-network-inspector/thumbnail.png
+---
+
+GraphQL Network Inspector is a dedicated devtools extension which allows you to inspect and debug
+GraphQL queries and mutations from network traffic.
+
+The extension has seen great adoption since its initial release and now I'm excited to announce it's
+new home as part of The Guild.
+
+You can check out the open source extension and try it out today.
+[GraphQL Network Inspector](https://github.com/warrenday/graphql-network-inspector)
```

Comment:
would you want to create a website before the release of the post or not?

Reply:
I'm putting a docs website together but I think we can release the post ahead and update when the site is ready?